### ISSUE TITLE
Workstation 0.5.1 - inspec error

### ISSUE THREAD

**Issue by kmf** [chef/chef-workstation#404]
Running `inspec` from chef-workstation, I get this error

```
WARN: Unresolved or ambigious specs during Gem::Specification.reset:
      bundler (>= 1.10)
      Available/installed versions of this gem:
      - 1.17.3
      - 1.17.2
WARN: Clearing out unresolved specs. Try 'gem cleanup <gem>'
Please report a bug if this causes problems.
```

When I run `inspec` from the normal package I don't get an error.

**Comment by btm** [comment]
This will be fixed soon in the next release by https://github.com/chef/chef-dk/pull/2184.

**Comment by ericcalabretta** [comment]
What's the workaround to use InSpec with Chef-Workstation? I'm in Chef Workstation 0.6.2 and have the identical error when I try to use InSpec.

**Comment by logan-pugh** [comment]
I think this issue is still present in Chef Workstation 0.6.2:

```
PS C:\opscode\chef-workstation\bin> chef --version
Chef Workstation version: 0.6.2
Chef Infra Client version: 15.1.36
Chef InSpec version: 4.7.3
Test Kitchen version: 2.2.5
Foodcritic version: 16.1.1
Cookstyle version: 5.0.0
PS C:\opscode\chef-workstation\bin> inspec --version
WARN: Unresolved or ambigious specs during Gem::Specification.reset:
      bundler (>= 1.10)
      Available/installed versions of this gem:
      - 1.17.3
      - 1.17.2
WARN: Clearing out unresolved specs. Try 'gem cleanup <gem>'
Please report a bug if this causes problems.
4.7.3
```

**Comment by marcparadise** [comment]
Thanks for the updated info.  We've identified the cause and are working to get a release out tomorrow.   

Unfortunately, there is no workaround for this.

**Comment by marcparadise** [comment]
Fixed in #413 and #412.

**Comment by Stromweld** [comment]
It's also present in when running test-kitchen. I'm on chef-workstation 0.6.2 as well.
```
Chef Workstation version: 0.6.2
Chef Infra Client version: 15.1.36
Chef InSpec version: 4.7.3
Test Kitchen version: 2.2.5
Foodcritic version: 16.1.1
Cookstyle version: 5.0.0
```
